apigw-http-api-cognito-lambda-cdk/example-pattern.json

```diff
@@ -0,0 +1,62 @@
+{
+    "title": "Amazon API Gateway HTTP API with Cognito JWT and Lambda integration",
+    "description": "This pattern create an Amazon Gateway API (v2) and two Lambda functions protected by JwtAuthorizer and Cognito for user management.",
+    "language": "Node.js",
+    "level": "200",
+    "framework": "CDK",
+    "introBox": {
+      "headline": "How it works",
+      "text": [
+        "This pattern creates an Amazon API Gateway HTTP API with JWTAuthorizer and Cognito as user management. It integrates with one public Lambda and one protected Lambda.."
+      ]
+    },
+    "gitHub": {
+      "template": {
+        "repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/apigw-http-api-cognito-lambda-cdk",
+        "templateURL": "serverless-patterns/apigw-http-api-cognito-lambda-cdk",
+        "projectFolder": "apigw-http-api-cognito-lambda-cdk",
+        "templateFile": ""
```

Comment:
Please update this

"templateFile": "lib/http-api-cognito-lambda-stack.ts"

Reply:
Done